packages/fxa-settings/src/components/GetDataTrio/en.ftl

```diff
@@ -3,12 +3,21 @@
 get-data-trio-title-firefox = { -brand-firefox }
 get-data-trio-title-firefox-recovery-key = { -brand-firefox } account recovery key
 get-data-trio-title-backup-verification-codes = Backup authentication codes
-get-data-trio-download-2 =
+
+# Since the buttons are of fixed width, Long label texts will be truncated.
```

Comment:
Good advice in this comment - just note that # comments are directly associated with the next string. E.g., this one would only be shown on Pontoon for `get-data-trio-download-v3` but not the following strings.

With the suggestion a bit lower down of splitting the string into separate ids for label and title, I would recommend tweaking (and repeating) the comments so they make sense in isolation with each string.